### ISSUE TITLE
Tanka deployment using obsolete k8s-library

### ISSUE THREAD

**Issue by c0ffeec0der** [grafana/agent#1590]
When generating manifest from tanka, this warning occur

TRACE: k8s-alpha is deprecated, please use 'github.com/jsonnet-libs/k8s-libsonnet' instead.

However when using the updated library, error occured

RUNTIME ERROR: couldn't open import "doc-util/main.libsonnet": no match locally or in the Jsonnet library paths

My analysis is that the libary use below strategy

```
local k = import 'ksonnet-util/kausal.libsonnet';

k + config { ...

$.util ..

or

$.core ...
```

loki tanka deployment is using function method call and upgrading k8s library have no issue; perhaps this libary could follow the same pattern; but will require quite a lot of revamp..

**Comment by rfratto** [comment]
Hi, sorry, which Tanka deployment are you using? I can only find references to k8s-alpha in our `example/k3d` folder, which isn't meant to be used for production or as a library.

**Comment by c0ffeec0der** [comment]
Hello,

I was referring to this page

https://github.com/grafana/agent/blob/main/production/README.md#use-our-production-tanka-configs

The recommendation is as below

```
# substitute your target k8s version for "1.16" in the next few commands
jb install github.com/jsonnet-libs/k8s-alpha/1.16
```

when I switch to github.com/jsonnet-libs/k8s-libsonnet ; it won't work as per my initial post

**Comment by rfratto** [comment]
Right, that page is definitely out of date and needs to be updated. Thanks for clarifying!

**Comment by vitalyu** [comment]
Still got this error! Any updates?

**Comment by rfratto** [comment]
At this point, we really haven't maintained the Tanka library, and it's probably likely to go away in the future because of that. 

I'd recommend trying to use the Grafana Agent [Helm chart](https://artifacthub.io/packages/helm/grafana/grafana-agent) with Tanka's native [support](https://tanka.dev/helm) for Helm charts. Helm charts are where we're investing more of our time in, since Helm is dramatically more popular than Jsonnet/Tanka.

**Comment by rfratto** [comment]
The Tanka library has been removed in favor of the Helm charts a few months ago. Closing as won't do.
